content/applications/inventory_and_mrp/purchase/manage_deals/blanket_orders.rst

```diff
@@ -1,204 +1,176 @@
-  Arrival* field on the :abbr:`RFQ (Request for Quotation)`.
-- :guilabel:`Source Document`: is the source purchase order (PO) that this blanket order will be
-  tied to. If this blanket order should not be tied to any existing :abbr:`POs (Purchase Orders)`,
-  leave this field blank.
+  value automatically populates the *Order Deadline* field on the |RfQ|.
+- :guilabel:`Delivery Date`: is the expected delivery date that the products included in an |RfQ|
+  are expected, if created directly from a blanket order form. If a new quotation is created, this
+  value automatically populates the *Expected Arrival* field on the |RfQ|.
+- :guilabel:`Source Document`: is the source purchase order (PO) that this blanket order is tied to.
+  If this blanket order should not be tied to any existing |PO|, leave this field blank.
 - :guilabel:`Company`: is the company assigned to this specific blanket order. By default, this is
   the company that the user creating the blanket order is listed under. If the database is not a
-  multi-company database, this field can not be changed, and will default to the only company
+  multi-company database, this field **cannot** be changed, and will default to the only company
   listed in the database.
 
 .. image:: blanket_orders/blanket-orders-new-agreement.png
    :align: center
    :alt: New blanket order purchase agreement with added products.
 
 Once all relevant fields have been filled out, click :guilabel:`Add a line` to add products under
-the :guilabel:`Product` column. Then, change the quantity of each product in the
-:guilabel:`Quantity` column (if desired), and set a price in the :guilabel:`Unit Price` column.
+the :guilabel:`Product` column. Then, in the :guilabel:`Quantity` column, change the quantity of
+each product, and set a price in the :guilabel:`Unit Price` column.
 
 .. important::
    When adding products to a new blanket order, the pre-existing prices of products will not be
-   added automatically to the product lines. The prices must be manually assigned by changing the
-   value in the :guilabel:`Unit Price` column to an agreed-upon price with the listed vendor.
-   Otherwise, the price will remain **0**.
+   automatically added to the product lines. Instead, the prices **must** be manually assigned, by
+   changing the value in the :guilabel:`Unit Price` column to an agreed-upon price with the listed
+   vendor. Otherwise, the price will remain **0**.
 
 To view and change the default purchase agreement settings for blanket orders directly from the
-blanket order form, click the :guilabel:`internal link (arrow icon)` next to the
-:guilabel:`Agreement Type` field where :guilabel:`Blanket Order` is listed. This navigates to the
-blanket order settings.
+blanket order form, click the :guilabel:`➡️ (internal link arrow)` next to the :guilabel:`Agreement
+Type` field where :guilabel:`Blanket Order` is listed. This navigates to the blanket order settings.
```

Comment:
since it was a little tricky to find this, what do you think about including an image of where this internal link arrow is? Totally optional!